getting-started.ipynb

```diff
@@ -29,28 +29,34 @@
   "cells": [
     {
       "cell_type": "markdown",
-      "source": [
-        "# Getting Started"
-      ],
       "metadata": {
-        "id": "HgljXEAJEcFq"
-      }
-    },
-    {
-      "cell_type": "markdown",
+        "id": "P9VjKOirEcFu"
+      },
       "source": [
+        "---\n",
+        "title: Getting Started\n",
+        "date: 2021-07-27\n",
+        "downloads: true\n",
+        "sidebar: true\n",
+        "tags:\n",
+        "  - PyTorch-Ignite\n",
+        "---\n",
```

Comment:
I can see still the frontmatter (https://colab.research.google.com/github/pytorch-ignite/examples/blob/frontmatter/getting-started.ipynb). It should be commented right as explained in #7 right?